### ISSUE TITLE
illegal hardware instruction

### ISSUE THREAD

**Issue by fenhl** [sindresorhus/dark-mode#13]
I get the following error message:

```
$ dark-mode status
fatal error: unexpectedly found nil while unwrapping an Optional value
Current stack trace:
0    dark-mode                          0x0000000106f52b20 swift_reportError + 132
1    dark-mode                          0x0000000106f6fed0 _swift_stdlib_reportFatalError + 61
2    dark-mode                          0x0000000106cddc90 specialized specialized StaticString.withUTF8Buffer<A> ((UnsafeBufferPointer<UInt8>) -> A) -> A + 355
3    dark-mode                          0x0000000106edff00 partial apply for (_fatalErrorMessage(StaticString, StaticString, StaticString, UInt, flags : UInt32) -> Never).(closure #2) + 109
4    dark-mode                          0x0000000106cddc90 specialized specialized StaticString.withUTF8Buffer<A> ((UnsafeBufferPointer<UInt8>) -> A) -> A + 355
5    dark-mode                          0x0000000106e9a880 specialized _fatalErrorMessage(StaticString, StaticString, StaticString, UInt, flags : UInt32) -> Never + 96
6    dark-mode                          0x0000000106c36d60 static DarkMode.isDark.getter + 334
7    dark-mode                          0x0000000106c376d0 main + 1450
8    libdyld.dylib                      0x00007fffeafa1234 start + 1
zsh: illegal hardware instruction  dark-mode status
$ dark-mode --version
2.0.1
```

The program takes a few minutes to produce the error message. `dark-mode on` also takes several minutes to run but doesn't produce any output and doesn't enable dark mode. Everything worked fine yesterday.

**Comment by fenhl** [comment]
I should probably clarify that this is on macOS 10.12.6.

**Comment by fenhl** [comment]
The error goes away after a reboot, but then reappears seemingly randomly after a while.

**Comment by anderscarling** [comment]
Code that caused this was seemingly removed in 40dbfce2d32a585a456051a5c0c043aab050834e (included in v3.0.0) so this issue should probably be closed.
